gax-java/gax/src/test/java/com/google/api/gax/retrying/AbstractRetryingExecutorTest.java

```diff
@@ -222,29 +222,29 @@ public void testTotalTimeoutExceeded() throws Exception {
   }
 
   @Test
-  public void testCancelOuterFutureBeforeStart() throws Exception {
-    FailingCallable callable = new FailingCallable(4, "request", "SUCCESS", tracer);
+  public void testCancelOuterFutureBeforeStart() {
+    // Use MockCallable instead of FailingCallable as we do not need the request to be re-tried.
+    // For this test, the callable should attempt to run and then see that the external future
+    // is cancelled and that the callable's execution has ended.
+    MockCallable callable = new MockCallable("SUCCESS");
```

Comment:
If we don't verify the tracer anymore, do we still need this new `MockCallable`? Can we keep using `FailingCallable`? I think the flakiness was only from `verifyNoMoreInteractions()`?